### PR TITLE
remove option --ostree enable

### PR DESCRIPTION
**Pull request by rabajaj0509**
`"--katello-enable-ostree true"` this option does not exists.

I get error as:
```
TASK [foreman_installer : Run installer] ***************************************
fatal: [centos7-luna-devel]: FAILED! => changed=true 
  cmd: |-
    foreman-installer -v --no-colors --disable-system-checks --scenario katello-devel --katello-devel-github-username 'rabajaj0509' --ka
tello-devel-enable-ostree=true --katello-devel-extra-plugins theforeman/foreman_remote_execution --katello-devel-extra-plugins theforema
n/foreman_discovery --katello-devel-extra-plugins theforeman/foreman_ansible --katello-devel-extra-plugins theforeman/foreman-tasks --ka
tello-devel-extra-plugins theforeman/foreman_bootdisk --katello-devel-extra-plugins theforeman/foreman_openscap --katello-devel-extra-pl
ugins theforeman/foreman_templates --katello-devel-extra-plugins theforeman/foreman_hooks --katello-devel-scl-ruby=rh-ruby25 --katello-d
evel-admin-password changeme --katello-devel-extra-plugins theforeman/foreman_remote_execution
  delta: '0:00:08.423099'
  end: '2021-03-03 13:48:12.373727'
  msg: non-zero return code
  rc: 1
  start: '2021-03-03 13:48:03.950628'
  stderr: |-
    ERROR: Unrecognised option '--katello-devel-enable-ostree'
  

```

### CODE REVIEW

File: playbooks/luna-devel.yml [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
-      - "--katello-devel-enable-ostree=true"
-      - "--katello-devel-extra-plugins theforeman/foreman_remote_execution"
-      - "--katello-devel-extra-plugins theforeman/foreman_discovery"
-      - "--katello-devel-extra-plugins theforeman/foreman_ansible"
-      - "--katello-devel-extra-plugins theforeman/foreman-tasks"
-      - "--katello-devel-extra-plugins theforeman/foreman_bootdisk"
-      - "--katello-devel-extra-plugins theforeman/foreman_openscap"
-      - "--katello-devel-extra-plugins theforeman/foreman_templates"
-      - "--katello-devel-extra-plugins theforeman/foreman_hooks"
+      - '--katello-devel-extra-plugins theforeman/foreman_remote_execution'
```
Comment (ekohl) [inline playbooks/luna-devel.yml:6]: Why are the different quotes needed?

Comment (rabajaj0509) [reply playbooks/luna-devel.yml:6]: blame my editor! it works fine with single quotes too, so i did not mind it. If you insist i can change it back :)

Comment (rabajaj0509) [reply playbooks/luna-devel.yml:6]: I see in github action, it says syntax error

**Comment by rabajaj0509** [comment]
Would you prefer to remove this option from other files too?

**Review by ekohl — COMMENTED** [review]
Yes, please also remove it from `playbooks/luna-devel.yml`.

For the others I'm not sure how much we want to keep it enabled on Katello 3.x releases.

Also, we can set `foreman_server_repositories_ostree` to false in most places (or stop including `ostree_repositories`).

File: pipelines/vars/forklift_katello.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ server_box:
   ansible:
     variables:
       foreman_server_repositories_katello: true
-      foreman_server_repositories_ostree: true
+      foreman_server_repositories_ostree: false
```
Comment (ekohl) [inline pipelines/vars/forklift_katello.yml:7]: I am worried this will break older pipelines

File: playbooks/bats_pipeline_katello_nightly.yml [hunk 3]
```diff
@@ -12,7 +12,7 @@
     foreman_installer_disable_system_checks: true
     foreman_installer_options_internal_use_only:
       - "--foreman-initial-admin-password {{ foreman_installer_admin_password }}"
-      - "--katello-enable-ostree=true"
+      - "--katello-enable-ostree=false"
```
Comment (ekohl) [inline playbooks/bats_pipeline_katello_nightly.yml:15]: This parameter no longer exists. I wonder if this playbook is still used.

**Review by ekohl — COMMENTED** [review]
@evgeni could you also have a look?

File: pipelines/vars/forklift_katello.yml (hunk 2 above)
Comment (evgeni) [reply pipelines/vars/forklift_katello.yml:7]: We don't run any ostree *tests*, so probably not. But also enabling those repos on current nightly has no harm, so I'd leave that `true` for now?

File: playbooks/bats_pipeline_katello_nightly.yml (hunk 3 above)
Comment (evgeni) [reply playbooks/bats_pipeline_katello_nightly.yml:15]: *probably* not.

`centos7-bats` box refers to `playbooks/bats_pipeline_nightly.yml` which doesn't exist, and `centos7-katello-bats-ci` (in ` 99-local.yaml.example`) refers to this one, but I don't think anyone uses those boxes today?

Comment (rabajaj0509) [reply playbooks/bats_pipeline_katello_nightly.yml:15]: Let keep this PR to remove the ostree parameter and I will create another PR to remove that playbook if required.

**Comment by rabajaj0509** [comment]
@evgeni @ekohl any further thoughts on this PR?

**Comment by evgeni** [comment]
there are more references, should these be killed/false'd too?
```
% git grep 'ostree.*true'
pipelines/vars/forklift_katello.yml:      foreman_server_repositories_ostree: true
pipelines/vars/forklift_katello.yml:      foreman_server_repositories_ostree: true
playbooks/foreman_proxy_content.yml:        - '--foreman-proxy-content-enable-ostree true'
playbooks/foreman_proxy_content_dev.yml:    foreman_server_repositories_ostree: true
playbooks/foreman_proxy_content_dev.yml:        - '--foreman-proxy-content-enable-ostree true'
```

**Review by ehelms — APPROVED** [review]

**Pull request merged**